pkgs/development/tools/misc/gede/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, fetchurl, ctags, qt4, python3}:
+
+stdenv.mkDerivation rec {
+
+  version = "2.0.3";
+  name = "gede-${version}";
+  src = fetchurl {
+    url = "http://gede.acidron.com/uploads/source/${name}.tar.xz";
+    sha256 = "1znlmkjgrmjl79q73xaa9ybp1xdc3k4h4ynv3jj5z8f92gjnj3kk";
+  };
+
+  buildInputs = [ ctags qt4 python3 ];
+  patches = [ ./build.patch ];
+
+  unpackPhase = ''
+    tar xf ${src}
+    cd ${name}
+  '';
+  configurePhase = "";
+  buildPhase = "";
+  installPhase = ''
+    ./build.py install --prefix=$out
```

Comment:
It seems to me that you ported this script from python2 to python3. I think it is better if you send your changes upstream and keep the patch minimal (only qmake-qt4 -> qmake) by using python2  here for the meantime.